public/app/features/explore/state/actions.ts

```diff
@@ -401,11 +431,304 @@ export function modifyQueries(
  */
 export function runQueries(exploreId: ExploreId): ThunkResult<void> {
   return (dispatch, getState) => {
-    dispatch(updateTimeRangeAction({ exploreId }));
-    dispatch(runQueriesAction({ exploreId }));
+    dispatch(updateTime({ exploreId }));
```

Comment:
As a note, not sure if this is because I already know what this should be doing, but I find this a bit more readable than before.